src/System.ComponentModel.TypeConverter/tests/Design/DesignerOptionServiceTests.cs

```diff
@@ -0,0 +1,487 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Xunit;
+
+namespace System.ComponentModel.Design.Tests
+{
+    public class DesignerOptionServiceTests
+    {
+        [Fact]
+        public void CreateOptionCollection_CreateMultipleTimes_ReturnsExpected()
+        {
+            var service = new TestDesignerOptionService();
+
+            DesignerOptionService.DesignerOptionCollection options1 = service.DoCreateOptionCollection(service.Options, "name", "value");
+            Assert.Equal(0, options1.Count);
+            Assert.NotEmpty(options1.Properties);
+            Assert.Same(options1.Properties, options1.Properties);
```

Comment:
Was this intentional? To check for the same property or did you mean? `Assert.Same(options1.Properties, service.Properties)`